src/_data/roadmap.json

```diff
@@ -60,16 +70,6 @@
       "description": "Easy dependency management and library upgrades.",
       "areas": ["unison-share", "ucm"]
     },
-    {
-      "title": "Graphical UCM",
```

Comment:
😢

Reply:
It's happening soon I think. But not literally right now. :)